packages/react-components/tsconfig.json

```diff
@@ -7,7 +7,8 @@
     "importHelpers": true,
     "jsx": "react",
     "noUnusedLocals": true,
-    "preserveConstEnums": true
+    "preserveConstEnums": true,
+    "resolveJsonModule": true
```

Comment:
What's the need for this?

Reply:
It is necessary to import `package.json` as a module. We use it to read repository.url for integration with GitHub, and to put the package version in the GitHub issue body.